### ISSUE TITLE
A problem about your profiling.

### ISSUE THREAD

**Issue by codinglzc** [ipftg/RTHMS#1]
When I run `make`, I get a error:
```
rthms_cache.h:1:30: fatal error: arpin_cachemodel.H: No such file or directory
 #include "arpin_cachemodel.H"
```
I can not find this file.
Please help me! Do you have more detailed documents about this profiling?

**Comment by Dwade3** [comment]
Excuse me. I have met the same problem. Have you solved this ?

**Comment by codinglzc** [comment]
No,I haven’t solved it. But I have deleted it and modified the code.

**Comment by Xiaoyang-Lu** [comment]
@codinglzc @Dwade3 I have the same problem. Can you install and use this tool now?
@ipftg Could you help me solve this problem?
